ci-operator/config/openshift/cincinnati/openshift-cincinnati-release-4.4.yaml

```diff
@@ -0,0 +1,133 @@
+base_images:
+  centos:
+    name: centos
+    namespace: ci
+    tag: "7"
+  openshift_release_golang-1.13:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+build_root:
+  image_stream_tag:
+    name: cincinnati-build-root
+    namespace: cincinnati-ci
+    tag: buildroot
+images:
+- dockerfile_path: dist/openshift-release/Dockerfile.builder
+  from: centos
+  to: buildroot
+- dockerfile_path: dist/Dockerfile.deploy/Dockerfile
+  from: centos
+  inputs:
+    buildroot:
+      as:
+      - quay.io/app-sre/cincinnati:builder
+  to: deploy
+- dockerfile_path: dist/Dockerfile.e2e/Dockerfile
+  from: centos
+  inputs:
+    buildroot:
+      as:
+      - quay.io/app-sre/cincinnati:builder
+    openshift_release_golang-1.13:
+      as:
+      - registry.svc.ci.openshift.org/openshift/release:golang-1.13
+  to: e2e-test
+promotion:
```

Comment:
No need to promote images in release-4.x - unless we want to use different build roots

Reply:
We might need to in case other branches advance in Rust toolchain versions we this branch can't for some reason